code/development-env/docker/README.md

```diff
@@ -19,18 +40,39 @@ docker-compose -f docker-compose-vault.yml -f docker-compose-mongo.yml up -d
 - Set Vault-App-Role token
 
 ```bash
-./scripts/configure-vault.sh
-export VAULT_APP_ROLE= # Set to value from line above
+../scripts/configure-vault.sh
+export VAULT_APP_ROLE= # Set to value output from command on line above
 ```
 
 - Start minikube proxy
 
 ```bash
-kubectl proxy
+kubectl proxy --address 0.0.0.0 --accept-hosts '.*'
 ```
 
-- Start DataLab APP/API, Infrastructure Api & Auth services
+- Start DataLab APP/API, Infrastructure Api and Auth services. If running on Mac or Linux, 
+replace `<platform>` with `mac` or `linux` as appropriate. Otherwise, remove 
+`-f docker-compose-app.<platform>.yml` from command.
 
 ```bash
-docker-compose -f docker-compose-app.yml up
+docker-compose -f docker-compose-app.yml -f docker-compose-app.<platform>.yml up
 ```
+
+
+## Update Mongo Record
+
+The default record that is created in the Mongo container needs to be updated so that 
```

Comment:
I wonder whether we should also just modify the data being inserted in the dummy record to include all of our user ids so we don't need to do this regularly.